recipes/BEHST/meta.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: behst
+  version: 29efb4c3a8e333a2824430ef32d8bf01
```

Comment:
My package does not have a version number  ( https://bitbucket.org/hoffmanlab/behst ). And I don't know how to set it...